addons/hr_timesheet/static/src/components/task_with_hours/task_with_hours.js

```diff
@@ -6,8 +6,8 @@ import { Many2OneField, many2OneField } from "@web/views/fields/many2one/many2on
 
 class TaskWithHours extends Many2OneField {
 
-    canCreate(props = this.props) {
-        return Boolean(props.context.default_project_id);
+    canCreate() {
+        return Boolean(this.context.default_project_id);
```

Comment:
You remove our fix. 😢 

In fact, it was expected but we should probably do it in another way to have the "new" context with the new props.

`canCreate` method is used in `computeActiveActions` method, line 45 in the same file and the props passed in that method is the new props (after a onWillUpdateProps), that's why we have to pass the props in parameter to be sure we use the new ones instead of the old ones.

Reply:
After discussing with @FrancoisGe , the actual fix done is right since the evalContext will correctly have the right context. Sorry. 😇